config/database.yml.sample

```diff
@@ -0,0 +1,86 @@
+# PostgreSQL. Versions 9.3 and up are supported.
```

Comment:
Could you remove the comments from this file and change the database prefix from "rails_bump" to "railsbump" please?